libs/design-system/brandmark/src/lib/brandmark.component.scss

```diff
@@ -0,0 +1,3 @@
+:host {
+  display: block;
```

Comment:
Add a css variable to control the color, `--hra-brandmark-color`. Default to `--sys-tertiary`